### PR TITLE
Add cron expression length validation

### PR DESCRIPTION
**Pull request by danielmkn**

### CODE REVIEW

File: validator/validator.go [hunk 1]
```diff
@@ -46,6 +46,21 @@ func Cron(value interface{}, _ cty.Path) diag.Diagnostics {
...
 }
 
+func CronLength(value interface{}, _ cty.Path) diag.Diagnostics {
+	var diags diag.Diagnostics
+	cron := regexp.MustCompile("[^\\s]+").FindAllString(value.(string), -1)
+
+	if len(cron) > 7 || len(cron) < 6 {
+		diags = append(diags, diag.Diagnostic{
+			Severity: diag.Error,
+			Summary:  "Invalid Cron expression, value should be between 6 and 7 characters long",
```
Comment (alexhung) [inline validator/validator.go:56]: s/characters/parts

File: validator/validator.go [hunk 2]
```diff
@@ -46,6 +46,21 @@ func Cron(value interface{}, _ cty.Path) diag.Diagnostics {
 	return diags
 }
 
+func CronLength(value interface{}, _ cty.Path) diag.Diagnostics {
+	var diags diag.Diagnostics
+	cron := regexp.MustCompile("[^\\s]+").FindAllString(value.(string), -1)
+
+	if len(cron) > 7 || len(cron) < 6 {
```
Comment (alexhung) [inline validator/validator.go:53]: Not a blocker, my preference is to write this like I'd read it. e.g. `if len(cron) < 6 || len(cron) > 7`

**Review by alexhung — CHANGES_REQUESTED** [review]

File: validator/validator.go (hunk 2 above)
Comment (danielmkn) [reply validator/validator.go:53]: okay :)

File: validator/validator.go [hunk 3]
```diff
@@ -46,6 +46,21 @@ func Cron(value interface{}, _ cty.Path) diag.Diagnostics {
 	return diags
 }
 
+func CronLength(value interface{}, _ cty.Path) diag.Diagnostics {
+	var diags diag.Diagnostics
+	cron := regexp.MustCompile("[^\\s]+").FindAllString(value.(string), -1)
```
[the remark this answers is not in the document]
Comment (danielmkn) [reply validator/validator.go]: yes

**Review by alexhung — APPROVED** [review]

**Review by Turhsus — COMMENTED** [review]
LGTM

**Pull request merged**